lib/packwerk/application_load_paths.rb

```diff
@@ -0,0 +1,65 @@
+# typed: strict
+# frozen_string_literal: true
+
+require "bundler"
+
+module Packwerk
+  module ApplicationLoadPaths
+    class << self
+      extend T::Sig
+
+      sig { returns(T::Array[String]) }
+      def extract_relevant_paths
+        assert_application_booted
+        all_paths = extract_application_autoload_paths
+        relevant_paths = filter_relevant_paths(all_paths)
+        assert_load_paths_present(relevant_paths)
+        relative_path_strings(relevant_paths)
+      end
+
+      sig { void }
```

Comment:
Add these as private methods?

Reply:
At least the filter method needs to be public so I can test it 🤷 

And I would love to list them in the order they're executed here.

Not sure making some of them private is worth it.

Or should I just circumvent privacy in the test? That doesn't seem ideal either.

Reply:
I think this is good.